src/main/PlagueMain.java

```diff
@@ -522,6 +570,10 @@ public void registerClientCommands(CommandHandler handler) {
             endgame(new Seq<>(), args);
         });
 
+        handler.<Player>register("monos", "explain monos", (args, player) -> {
+            Call.infoMessage(player.con, mono_info);
+        });
+
```

Comment:
bundle into help

Reply:
why though? when people say, why mono explode, if you tell them to run `/info`, they're not going to read the entire thing to the end?
its easier to tell them to run `/monos`.

Reply:
fair enough.

Reply:
I bundled it into /info while keeping /monos.